AudioDelayFeedback.h

```diff
@@ -290,19 +299,24 @@ class AudioDelayFeedback
 		= coeff * (in-last_out) + last_in
 		*/
 		//setPin13High();
+
+	  /* I think these should **not** be static 
 		static int8_t last_in;
```

Comment:
These two were static originally. I think that prevents this class from being used as several instances, as they would share `last_in` and `last_out`, but maybe I am being mistaken?

Reply:
I'm not sure, Tom.  Definitely worth checking.
They're inside a function definition and in that context they persist between function calls, but are inaccessible from outside.  They can also have a speed benefit if cached locally.  
If they were static variables in the class outside the function, you're right, they would be shared between all objects of that class and also accessible from outside the class with ::, without even needing an object to be instantiated.  (I just googled this yesterday for something else - it's probably worth checking).  
Also, the example AudioDelayFeedbackX2 seems to work...?

Reply:
Aha, you're right (as usual).
It will be interesting to hear what the Mozzi example sounds like when it works properly!

Reply:
Will test nevertheless. I was initially confused not to see any impact on the reports by actually these examples are not compiled in the autotests.